DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,21 @@
    "metadata": {},
```

Comment:
<p>Can we just add a note here saying "These observations are derived using various methods, in particular using surface flux observations and method 2 (which assumes a steady state)".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>